.github/actions/setup-pnpm/action.yml

```diff
@@ -17,6 +17,7 @@ runs:
       with:
         node-version: ${{ inputs.node_version }}
         cache: pnpm
+        registry-url: "https://registry.npmjs.org"
```

Comment:
By specifying `registry-url`, setup-node creates `.npmrc` that refers NODE_AUTH_TOKEN. [(doc)](https://github.com/actions/setup-node#:~:text=Optional%20registry%20to%20set%20up%20for%20auth)